tests/test_models.py

```
@@ -2,772 +2,228 @@
+class TestModel(ValidationMixin, UnifiBaseModel):
+    """Test model with validation."""
+
+    mac: str = Field(description="MAC address")
+    ip: Optional[str] = Field(None, description="IP address")
+    netmask: Optional[str] = Field(None, description="Network mask")
+    mac_list: Optional[List[str]] = Field(None, description="MAC address list")
+    version: Optional[str] = Field(None, description="Version string")
+
+
+@pytest.mark.parametrize(
+    "field,value,expected_value,description",
```

Ruff (0.8.2)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)